region/lowernorfair/east.json

```diff
@@ -1225,19 +1318,107 @@
           "to": [
             {
               "id": 1,
-              "requires": [ "canUsePowerBombs" ],
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                "canUsePowerBombs",
+                {"cost": [
+                  {"heatFrames": 600},
+                  {"ammo": [{
+                    "type": "PowerBomb",
+                    "count": 1
+                  }]}
+                ]}
+              ],
               "unlock": null
             },
             {
               "id": 2,
-              "requires": [ "canPassBombPassages" ],
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                "Morph",
+                {"or": [
+                  "Bombs",
+                  {"and": [
+                    "PowerBomb",
```

Comment:
There seems to be some inconsistency on whether the item itself is required alongside its ammo cost. It was handled differently with Supers in writg. Any opinion on which one it should be?

Reply:
Adding Super ammo in writg was kind of a mistake. I will remove super ammo and add "Super" req. Will need some kind of way to handle these big uses of ammo though.

Reply:
IIRC there was the idea of using an `enemyKill` in costs, which we discussed during preCathedral, but it wasn't clear how it should work.

Reply:
> Have you considered the application of `canHitBox` in Red Kihunter Shaft?

Going down from the top it only helps about 15 frames. Going up it's too inconsistent with the RNG kihunters placement sometimes lurking straight above etc.